mysite/templates/BlogCommentEmail.ss

```diff
@@ -1,403 +0,0 @@
-                              <% end_if %>
-                              <% if $ApproveLink %>
-                              <table class="button approve" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 100%; overflow: hidden; padding: 0;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td style="word-break: break-word; -webkit-hyphens: auto; -moz-hyphens: auto; hyphens: auto; border-collapse: collapse !important; vertical-align: top; text-align: center; color: #ffffff; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; display: block; width: auto !important; background: #005B95; margin: 0; padding: 8px 0; border: 1px none #2284a1;" align="center" bgcolor="#005B95" valign="top">
-                                  <a href="$ApproveLink" style="color: #ffffff; text-decoration: none; font-weight: bold; font-family: Helvetica, Arial, sans-serif; font-size: 16px; display: block;">Approve it</a>
-                                </td>
-                              </tr></table>
-                              <% end_if %>
-                              </td>
-                          <td class="expander" style="word-break: break-word; -webkit-hyphens: auto; -moz-hyphens: auto; hyphens: auto; border-collapse: collapse !important; vertical-align: top; text-align: left; visibility: hidden; width: 0px; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 0;" align="left" valign="top"></td>
-                        </tr></table></td>
-                  </tr></table>
-                  <% else %>
-                  <table class="row footer" style="border-spacing: 0; border-collapse: collapse; vertical-align: top; text-align: left; width: 100%; position: relative; display: block; padding: 0px;"><tr style="vertical-align: top; text-align: left; padding: 0;" align="left"><td class="wrapper" style="word-break: break-word; -webkit-hyphens: auto; -moz-hyphens: auto; hyphens: auto; border-collapse: collapse !important; vertical-align: top; text-align: left; position: relative; color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; line-height: 19px; font-size: 14px; margin: 0; padding: 10px 20px 0px 0px;" align="left" valign="top">
-
-                      <p style="color: #222222; font-family: 'Helvetica', 'Arial', sans-serif; font-weight: normal; text-align: left; line-height: 19px; font-size: 14px; margin: 0 0 10px; padding: 0;" align="left">You can view or moderate this comment at <a href="{$Comment.Link}" style="color: #005B95; text-decoration: none;">{$Parent.Title}</a></p>
-
-                    </td>
-                  </tr></table>
-                  <% end_if %>
-                  </td>
-              </tr></table></center>
-        </td>
-      </tr></table></body>
-</html>
```

Comment:
Similar to earlier with the config blocks - do we need to add this again somewhere else?

Reply:
`cwp-recipe-blog` I'd imagine.

Reply:
`silverstripe/recipe-blog` as it turns out :)